frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -121,6 +115,7 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
           <LogsTabStatus
             loaded={loaded}
             error={error}
+            isLogsAvailable={podContainers.length !== 1 && logs !== ''}
```

Comment:
I feel the logic is not correct here... if you simply do `!(podContainers.length === 1 && !logs)` you will get `podContainers.length !==1 || !!logs`